src/locale/layout/organize/en.yml

```diff
@@ -72,6 +72,9 @@ surveys:
   actions:
     publish: Publish survey
     unpublish: Unpublish survey
+  stats:
+    questions: '{numQuestions, plural, one {1 question} other {# questions}}'
+    submissions: '{numSubmissions, plural, one {1 submission} other {# submissions}}'
```

Comment:
These are under `surveys` (line 71), which is correct. This is why `layout.organize.survey` should not be needed (see previous comment).